### PR TITLE
Fix Aws api Gateway "How it works See it in Action" Component

### LINKED ISSUE #4851: [Screenshots] AWS API Gateway Operator under 'How it Works See It in Action' section
### Current Behavior
Currently, [every integration](https://layer5.io/cloud-native-management/meshery/integrations/aws-api-gateway-operator) has only two old screenshots of the designer and visualizer in the 'How it Works See It in Action' section.

### Desired Behavior
- Add new images and videos from the latest version of the software ([MeshMap](https://layer5.io/cloud-native-management/meshmap))
- Add screenshots for [AWS API Gateway Operator](https://layer5.io/cloud-native-management/meshery/integrations/aws-api-gateway-operator) integration.

### Acceptance Tests
- Ensure your screenshot represents the use of a few specific integrations.
- The screenshot from the Designer mode should contain a valid design, refer to [meshery catalog](https://meshery.io/catalog) to use and view published designs.
- Add both Dark Mode and Light Mode screenshots.
- The screenshot should not be a snippet, take the full screenshot of your browser.
- Use Incognito mode on Chrome browser, to ensure your bookmark bar or any other items are not shown in the screenshot/video.

---

<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Cvr421**
**Description**
Adding the New Meshmap Designer and visualizer screenshot in [Aws api gateway operator](https://layer5.io/cloud-native-management/meshery/integrations/aws-api-gateway-operator)
This PR fixes #4851

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- 

- Yes, I signed my commits.

### CODE REVIEW

**Comment by Cvr421** [comment]
Hii @saurabh100ni Please review this PR . And let me know your thoughts if anything need to change.

**Comment by l5io** [comment]
🚀 Preview for commit e0ad9ec04ef12ba976b3ab47cf21a3ad4d7f6308 at: https://654143df3e1e553c746d9458--layer5.netlify.app

**Comment by saurabh100ni** [comment]
@sudhanshutech //

**Comment by leecalcote** [comment]
@vishalvivekm how are you ensuring that these are not being overwritten?

**Comment by saurabh100ni** [comment]
Please add this as your agenda in [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) to present it in the Website Meet today at 6:30 pm IST/7:00 am CT

**Comment by l5io** [comment]
🚀 Preview for commit d5f94e418dcc6e7196d634f87b2d7bba067af6e7 at: https://655e00e4fd1a4128ed3e6e15--layer5.netlify.app

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
@Cvr421 
Please create a `aws-api-gateway-operator` directory in integrations/_images/ and move all your screenshots in that. I mean doing this: 
```
        renamed:    aws-api-Design-dark.png -> aws-api-gateway-operator/aws-api-Design-dark.png
        renamed:    aws-api-Design.png -> aws-api-gateway-operator/aws-api-Design.png
        renamed:    aws-api-visualizer-dark.png -> aws-api-gateway-operator/aws-api-visualizer-dark.png
        renamed:    aws-api-visualizer.png -> aws-api-gateway-operator/aws-api-visualizer.png
 ```

File: src/collections/integrations/aws-api-gateway-operator/index.mdx [hunk 1]
```diff
@@ -11,8 +11,11 @@ featureList: ["Create, publish, maintain, monitor, and secure REST, HTTP, and We
 "API Gateway creates RESTful APIs that enable stateless client-server communication."
 ]
 workingSlides: [
-    ../_images/meshmap-visualizer.png,
-    ../_images/meshmap-designer.png]
+    ../_images/aws-api-Design.png,
+    ../_images/aws-api-Design-dark.png,
+    ../_images/aws-api-visualizer.png,
+    ../_images/aws-api-visualizer-dark.png
+    ]
```
Comment (vishalvivekm) [inline src/collections/integrations/aws-api-gateway-operator/index.mdx:18]: please revert this change after moving screenshots you've uploaded to its specific directory.
@Cvr421

Comment (vishalvivekm) [reply src/collections/integrations/aws-api-gateway-operator/index.mdx:18]: Refer to this: https://github.com/layer5io/layer5/pull/5129#issue-1995493031

**Comment by Cvr421** [comment]
@vishalvivekm I have made the changes requested by you . Please review

**Comment by l5io** [comment]
🚀 Preview for commit bb4d5af4e9935c0c638631889a31327a899b2630 at: https://6560c828cc5ae73d037d8df3--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 77640239c18135061cdbe4015c20bb7bb6d22a8f at: https://6560df741fd7770b2f12abcd--layer5.netlify.app

**Comment by vishalvivekm** [comment]
> @vishalvivekm I have made the changes requested by you . Please review And there are other commits is also merged with this branch i think. Should i need to do something for this OR it is ok

@Cvr421 signoff is missing. The changes are ready except the missing DCO. Please fix it.

**Comment by l5io** [comment]
🚀 Preview for commit 59b49fdfa5d70c3f0f129829c5d82dc27e2c871c at: https://65622732faa0394a9346eccc--layer5.netlify.app

**Comment by Cvr421** [comment]
@vishalvivekm i have added signoff and also fix the merge conflict please review .

**Review by sandramsc — COMMENTED** [review]
LGTM!

**Comment by leecalcote** [comment]
Good work!

**Pull request merged**